ui-react/src/components/topBar/LogsCanvas.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Offcanvas from "react-bootstrap/Offcanvas";
+
+type LogsCanvasProps = {
+  show: boolean;
+  handleClose?: () => {};
+};
+
+export default function LogsCanvas({ show, handleClose }: LogsCanvasProps) {
+  return (
+    <Offcanvas
+      show={show}
+      onHide={handleClose}
+      placement="bottom"
+      style={{ zIndex: 9999 }}
+    >
+      <Offcanvas.Header closeButton>
+        <Offcanvas.Title>לוגים:</Offcanvas.Title>
+      </Offcanvas.Header>
+      <Offcanvas.Body>
+        [2022-12-07 12:34:27.432] [info] Welcome to caspion log
+      </Offcanvas.Body>
```

Comment:
You missed fetching the real log here, right?

Reply:
I wanted to consult with you about this,
As I understand, currently the Logger is missing in the ui-react src folder
so we need to add it and start using it there. (correct me if I'm wrong )

and if so, maybe it should be on another PR ?

Reply:
The *Vue* code is using the `logger.ts` file, I don't see a reason to not use it in *React*.

Maybe the challenge is using the same code in a different project, now when the backend is separated from the `react-ui`. @brafdlog What do you suggest for this case?